apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -437,12 +435,23 @@ all_accounts_balances(_Config) ->
     rpc(application, set_env, [aehttp, enable_debug_endpoints, true]),
     GenesisAccounts = rpc(aec_genesis_block_settings, preset_accounts, []),
     BlocksToMine = 10,
+    Receivers = 10,
+    Fee = 1,
     aecore_suite_utils:mine_blocks(aecore_suite_utils:node_name(?NODE),
                                    BlocksToMine),
+    ReceiversAccounts = [{random_hash(), Idx} || Idx <- lists:seq(1, Receivers)],
+    lists:foreach(
+        fun({ReceiverPubKey, AmountToSpent}) ->
+            {ok, 200, _} = post_spend_tx(ReceiverPubKey, AmountToSpent, Fee)
+        end,
+        ReceiversAccounts),
+
+    % mine a block to include the txs
```

Comment:
I expect this test to eventually show to be fragile, as the mempool is not checked for inclusion of all txs.

Reply:
I've refactor the test to _exclude_ the public addresses of the recipients for any transaction that didn't make it into the block and was left in the mempool.

Reply:
I see potentially all txs may have been left in mempool.